emhttp/plugins/dynamix/DeviceInfo.page

```diff
@@ -770,7 +770,7 @@ _(Critical disk utilization threshold)_ (%):
 <form markdown="1" method="POST" action="/update.php" target="progressFrame">
 
 _(pool device stats)_:
-: <?echo "<pre>".shell_exec("/sbin/btrfs dev stats -T /mnt/$tag")."</pre>"?>
+: <?echo "<pre>" . htmlspecialchars(shell_exec("/sbin/btrfs dev stats -T /mnt/$tag"), ENT_QUOTES, 'UTF-8') . "</pre>"; ?>
```

Comment:
Please escape all vars being passed to shell_exec:
`shell_exec("/sbin/btrfs dev stats -T ".escapeshellarg("/mnt/$tag"))`

Reply:
Resolved.